packages/appium/docs/scripts/build-reference.js

```diff
@@ -0,0 +1,33 @@
+const {fs} = require('@appium/support');
+const {log, LANGS, DOCS_DIR} = require('./utils');
+const path = require('path');
+const monorepoRoot = path.resolve(__dirname, '..', '..', '..', '..');
+const typedocOut = path.resolve(monorepoRoot, 'typedoc-docs');
```

Comment:
probably more robust to grab this from the typedoc config file 

```suggestion
const {out: typedocOut} = JSON.parse(fs.readFileSync(path.join(monorepoRoot, 'typedoc.json')));
```